docs/_data/navs.yml

```diff
@@ -17,6 +17,8 @@ v1.0.x:
   - label: Pipeline hooks
   - label: The deployment context
   - label: Creating a plugin pack
+  - label: Creating an in-repo-addon plugin
```

Comment:
I set a shorter path because the title ended up being so long and I figured the chances of it changing are slightly higher than others because of its length. Any suggestions here?